[2025-06-10T15:38:42.922+0200] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-10T15:38:42.927+0200] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: case2.wait_for_taxi_zone_file manual__2025-06-10T13:38:35.812440+00:00 [queued]>
[2025-06-10T15:38:42.929+0200] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: case2.wait_for_taxi_zone_file manual__2025-06-10T13:38:35.812440+00:00 [queued]>
[2025-06-10T15:38:42.929+0200] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-06-10T15:38:42.934+0200] {taskinstance.py:2890} INFO - Executing <Task(FileSensor): wait_for_taxi_zone_file> on 2025-06-10 13:38:35.812440+00:00
[2025-06-10T15:38:42.937+0200] {standard_task_runner.py:72} INFO - Started process 30804 to run task
[2025-06-10T15:38:42.940+0200] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'case2', 'wait_for_taxi_zone_file', 'manual__2025-06-10T13:38:35.812440+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/case2.py', '--cfg-path', '/var/folders/sj/kmk_wl3517d377606n_z382h0000gp/T/tmp_0xxhcbt']
[2025-06-10T15:38:42.941+0200] {standard_task_runner.py:105} INFO - Job 28: Subtask wait_for_taxi_zone_file
[2025-06-10T15:38:42.981+0200] {task_command.py:467} INFO - Running <TaskInstance: case2.wait_for_taxi_zone_file manual__2025-06-10T13:38:35.812440+00:00 [running]> on host LGLXN9TVJX
[2025-06-10T15:38:43.013+0200] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='case2' AIRFLOW_CTX_TASK_ID='wait_for_taxi_zone_file' AIRFLOW_CTX_EXECUTION_DATE='2025-06-10T13:38:35.812440+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-10T13:38:35.812440+00:00'
[2025-06-10T15:38:43.014+0200] {taskinstance.py:732} INFO - ::endgroup::
[2025-06-10T15:38:43.098+0200] {baseoperator.py:423} WARNING - FileSensor.execute cannot be called outside TaskInstance!
[2025-06-10T15:38:43.105+0200] {base.py:84} INFO - Retrieving connection 'fs_default'
[2025-06-10T15:38:43.105+0200] {filesystem.py:109} INFO - Poking for file /Users/tsimsek/airflow/dags/../data/raw/zones_13062025.csv
[2025-06-10T15:38:43.106+0200] {filesystem.py:113} INFO - Found File /Users/tsimsek/airflow/dags/../data/raw/zones_13062025.csv last modified: 20250610152724
[2025-06-10T15:38:43.106+0200] {base.py:339} INFO - Success criteria met. Exiting.
[2025-06-10T15:38:43.107+0200] {filesystem.py:109} INFO - Poking for file /Users/tsimsek/airflow/dags/../data/raw/zones_13062025.csv
[2025-06-10T15:38:43.107+0200] {filesystem.py:113} INFO - Found File /Users/tsimsek/airflow/dags/../data/raw/zones_13062025.csv last modified: 20250610152724
[2025-06-10T15:38:43.110+0200] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-06-10T15:38:43.110+0200] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=case2, task_id=wait_for_taxi_zone_file, run_id=manual__2025-06-10T13:38:35.812440+00:00, execution_date=20250610T133835, start_date=20250610T133842, end_date=20250610T133843
[2025-06-10T15:38:43.158+0200] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-10T15:38:43.173+0200] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-10T15:38:43.174+0200] {local_task_job_runner.py:245} INFO - ::endgroup::
